Onboarding/TechicalOverview.md

```diff
@@ -0,0 +1,30 @@
+# CanDIG Technical Onboarding
+## CanDIG Project Overview
+The CanDIG project has a distributed architecture, meaning each site is meant to run their own _____(What do you call everything represented in the diagram together) and there is no centralized CanDIG server. Users can use their local CanDIG _____(^) to communicate with other sites. Every site is responsible for handling the authentication of it's users and the authorization of incoming requests from other sites. CanDIG uses an API driven approach, where users send requests to their CanDIG which communicate with other sites through ...?
+## Component Overview:
+### Tyk [Gateway, Pump]
+Tyk Gateway is an API managment platform that CanDIG uses to map API endpoints to endpoints in CanDIG server. When the user makes a request, it is made to Tyk, which forwards the request to the mapped endpoint in CanDIG server.  
+Tyk Pump collects metrics on the use of Gateway and has many methods for transmiting these metrics to various databases.
+### Keycloak
+Authentication. That's it.?
+### CanDIG server
+Fasciliates the communication between different sites running CanDIG. What else?
+### mongodb, redis
+Databases. What do they store? Absolutely no idea. Maybe this is what pump goes to?
```

Comment:
https://tyk.io/docs/planning-for-production/redis-mongodb/ says mostly analytics. I will leave it your google skills to figure out why two and what do they offer. Do not spend a whole day on it but it will be a nice learning experience to figure out what they each are good for, and how it relates to Tyk here. :slightly_smiling_face: